Readme.md

```diff
@@ -51,7 +51,15 @@ Configure the built-in AI options as detailed in the [AI docs](https://docs.micr
 
 ```
 
-In a multi-tenant setup you can configure different instrumentation keys to collect request tracking and client-side tracking data on different tenants, just follow [the Orchard Core configuration docs](https://docs.orchardcore.net/en/latest/docs/reference/core/Configuration/).
+Add the AI services to the service collection in your Web project's Program.cs file:
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+var configuration = builder.Configuration;
+
+builder.Services.AddOrchardCoreApplicationInsightsTelemetry(configuration)
+```
```

Comment:
```suggestion
```

Note that due to how the Application Insights .NET SDK works, telemetry can only be collected for the whole app at once; collecting telemetry separately for each tenant is not supported.
```